pytorch_blade/torch_blade/mlir/disc_engine_conversion.py

```diff
@@ -105,7 +105,9 @@ def _get_mlir_unsupported(graph):
     return unspt_nodes + extra_unspt_nodes
 
 def _disc_engine_conversion(module):
-    def try_cvt_to_disc_engine_func(c_module, subgraph, group_name, q_info=None):
+    def try_cvt_to_disc_engine_func(
+            c_module, subgraph, group_name, q_info=None, grp_calib_data=None
```

Comment:
Nit: grp_calib_data is not used for now here. Add it later when we support quantization in disc.

Reply:
disc and trt share the same `group_node_to_engine`. If `grp_calib_data` is not added, there will be a "argument number mismatch" problem